Replicator/tests/data/docker/README.md

```diff
@@ -0,0 +1,69 @@
+This docker folder contains the `docker-compose.yml` file for starting and configuring Couchbase Server and Sync Gateways for running non-walrus replicator tests with Sync Gateway.
+
+When running `docker compose up`, there will be one Couchbase Server and two Sync Gateways started as follows:
+
+### 1. Sync Gateway for .SyncServerCollection Tests
+
+ | Config      | Value       |
+ | ----------- | ----------- |
+ | Database    | scratch     |
+ | Port        | 4984        |
+ | Admin Port  | 4985        |
+ | Collections | flowers.roses, flowers.tulips, and flowers.lavenders |
+
+ #### Sync Function
+ ```JS
+    function (doc, oldDoc) { 
+        if (doc.isRejected == 'true') 
+            throw({'forbidden':'read_only'}); 
+        channel(doc.channels); 
+    }
+ ``` 
+
+### 2. Legacy Sync Gateway for .SyncServer30 Tests
+
+ | Config      | Value       |
+ | ----------- | ----------- |
+ | Database    | scratch-30  |
+ | Port        | 4884        |
+ | Admin Port  | 4885        |
+ | Collections | default     |
```

Comment:
Does SB 3.0 know Collections?

Reply:
No, it's just the name in the table here to make it consistent with the other table. I could remote it if that is confused.

Reply:
I think it's confusing. Perhaps put a note there in lieu of "default", something "N/A"